app/component/SummaryPage.js

```diff
@@ -2450,6 +2450,9 @@ class SummaryPage extends React.Component {
               />
             </div>
           );
+          // EMBARK override content with activeTab only
+          content = itineraryTabs[activeIndex];
```

Comment:
Would it work to just call the `selectedItineraries` map function with `selectedItiniraries[activeIndex]` & `activeIndex`, in order not to render all other tabs? We would still keep the diff against upstream relatively small.